src/main/java/it/gov/pagopa/payment/service/MerchantTransactionServiceImpl.java

```diff
@@ -61,7 +61,9 @@ public MerchantTransactionsListDTO getMerchantTransactions(String merchantId, St
                                     transaction.getTrxDate().toLocalDateTime(),
                                     authorizationExpirationMinutes,
                                     transaction.getUpdateDate(),
-                                    transaction.getStatus().toString()
+                                    transaction.getStatus().toString(),
```

Comment:
if inside this application this field is an enum, why inside the DTO returned you are change it into String?

Reply:
I didn't modify this field, in the DTO it was already present in the form of String